src/base/cacheManagerKeyv.ts

```diff
@@ -142,6 +49,99 @@ export class CacheManagerKeyv<T> implements ICacheManager {
     this.keyv.on("error", sdkLogger.error);
   }
 
+  addCacheHeaders = (resHeaders, path, key, hash, time): void => {
+    resHeaders.set(customCacheHeaders.localCache, encodeURIComponent(path));
+    resHeaders.set(customCacheHeaders.localCacheKey, encodeURIComponent(key));
+    resHeaders.set(customCacheHeaders.localCacheHash, encodeURIComponent(hash));
+    resHeaders.set(
+      customCacheHeaders.localCacheTime,
+      new Date(time).toUTCString()
```

Comment:
`new Date(invalid).toUTCString()` will return the string `"Invalid Date"` instead of throwing. Is that behavior handled anywhere? Does it need to be?

Reply:
Since this is existing code and this PR is small, I'm going to leave this alone. Especially because the headers are meant to be informational and we wouldn't want a problem here to be fatal. The existing behavior seems fine to me.